JavaScript/Topics/4.Command_Line/README.md

```diff
@@ -1 +1,75 @@
-# To Do
\ No newline at end of file
+# Introduce JavaScript's usage in the Command-Line 
+<br>
+
+***JavaScript*** is a powerful, widely used, open-source, and dynamic programming language used to add and render interactivity on websites. Also, a variety of other developer tools have been built on top of it, giving rise to a vast range of functionalities that can be implemented with ease to add on the dynamism of the website.<br>
+
+Besides using JS to render user-friendly interactive functionalities and features on our websites, a JavaScript shell (Command-line) enables us to quickly run and test snippets of JS code without having to load and reload it in a web page. This makes developing and debugging JS code extremely easy. These JS program that run on Command Line Interfaces (CLI) are the standalone JS scripts.
+<hr>
+Some JavaScript shells that are stand-alone environments are:
+
+- ***NodeJS*** - Node.js is a platform for easily building fast, scalable network applications.
```

Comment:
It's **Node.js**

Reply:
```suggestion
- ***NodeJS*** - Node.js is an open-source, single threaded JavaScript runtime for servers which puts chrome's V8 engine into your servers in order to execute JavaScript code outside your chrome browser.
```

Reply:
Updated the ReadMe with requested changes. Thank you. @dakshsethi